src/transforms/remap.rs

```diff
@@ -87,6 +89,7 @@ impl Remap {
             timezone: config.timezone,
             drop_on_error: config.drop_on_error,
             drop_on_abort: config.drop_on_abort,
+            enrichment_tables: enrichment_tables.as_search(),
```

Comment:
Why call `as_search` here, instead of requiring an owned `TableSearch` object in the `new` function signature? That seems to make more sense to me, but perhaps there's a technical reason why that's not possible.